tests/gluon_autolog/test_gluon_autolog.py

```diff
@@ -15,8 +15,6 @@
 import mlflow
 import mlflow.gluon
 
-client = mlflow.tracking.MlflowClient()
```

Comment:
The main change needed for all these per-framework tests was to compute the client within the test, after the tracking URI mock was already instantiated, instead of globally